migrations/lib/converters/discourse/settings.yml

```diff
@@ -7,3 +7,6 @@ source_db:
   user: "username"
   password: "password"
   dbname: "discourse_source_db"
+
+categories:
+  skip_existing_ids: false
```

Comment:
I think that the setting name is misleading. At first, I thought it will skip categories, but I guess the intent is to either merge into existing seeded categories or create new categories during import. Maybe `map_seeded_categories` or something along those lines?

Reply:
Yep, I agree.